lib/scoring/aggregate.js

```diff
@@ -24,6 +24,10 @@ function calculateAggregation(evaluations) {
     const accumulator = mapValues(shape, (value, key) => {
         return evaluations
         .map((evaluation) => objGet(evaluation, key))
+        // DownloadAcceleration is the only with negative values
```

Comment:
Can we remove this line and put `(e.g.: downloads acceleration) in the line below?